kde-frameworks/kapidox/kapidox-5.21.0.ebuild

```diff
@@ -4,16 +4,16 @@
 
 EAPI=6
 
-KDE_DEBUG="false"
-KDE_DOXYGEN="false"
-KDE_TEST="false"
 DISTUTILS_SINGLE_IMPL="true"
 PYTHON_COMPAT=( python2_7 )
-inherit kde5 distutils-r1
```

Comment:
https://qa-reports.gentoo.org/output/genrdeps/dindex/kde-frameworks/kapidox All consumers need cmake. What is the purpose to drop the dependency?